### PR TITLE
add_sort_ numpy_Frontend#3577

### PR DESCRIPTION
**Pull request by Hellenn07**

### CODE REVIEW

File: ivy/functional/backends/numpy/creation.py [hunk 1]
```diff
@@ -137,7 +137,11 @@ def full_like(
     device: str,
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
+ numpy_sort
+    # _assert_fill_value_and_dtype_are_compatible(dtype, fill_value)
+
     ivy.assertions.check_fill_value_and_dtype_are_compatible(fill_value, dtype)
+ master
```
Comment (ogbanugot) [inline ivy/functional/backends/numpy/creation.py:144]: Hi @Hellenn07, can you please take a look at these lines, it appears the merge from master left some text that shouldn't be in the code. Kindly fix these and request for a review once you are done thanks! 🙂

File: ivy/functional/backends/tensorflow/creation.py [hunk 2]
```diff
@@ -237,7 +237,11 @@ def full_like(
     device: str,
     out: Optional[Union[tf.Tensor, tf.Variable]] = None,
 ) -> Union[tf.Tensor, tf.Variable]:
+ numpy_sort
+    # _assert_fill_value_and_dtype_are_compatible(dtype, fill_value)
+
     ivy.assertions.check_fill_value_and_dtype_are_compatible(fill_value, dtype)
+master
```
Comment (ogbanugot) [inline ivy/functional/backends/tensorflow/creation.py:244]: here too

File: ivy/functional/backends/torch/creation.py [hunk 3]
```diff
@@ -247,7 +247,11 @@ def full_like(
     device: torch.device,
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
+numpy_sort
+    # _assert_fill_value_and_dtype_are_compatible(dtype, fill_value)
+
     ivy.assertions.check_fill_value_and_dtype_are_compatible(fill_value, dtype)
+master
```
Comment (ogbanugot) [inline ivy/functional/backends/torch/creation.py:254]: same here

File: ivy/functional/backends/torch/elementwise.py [hunk 4]
```diff
@@ -578,6 +578,22 @@ def subtract(
 subtract.support_native_out = True
 
 
+def isscalar(x):
+    # print("dim: {}".format(x.dim()))
+    # print("size: ", x.size())
+    # print("len: ", len(x))
```
Comment (ogbanugot) [inline ivy/functional/backends/torch/elementwise.py:584]: Also, can you remove the commented `print` statements.

File: ivy/functional/backends/torch/elementwise.py [hunk 5]
```diff
@@ -586,6 +602,41 @@ def remainder(
     modulus: bool = True,
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
+ numpy_sort
```
Comment (ogbanugot) [inline ivy/functional/backends/torch/elementwise.py:605]: see here again

File: ivy/functional/backends/torch/elementwise.py [hunk 6]
```diff
@@ -594,6 +645,7 @@ def remainder(
         diff, x2 = ivy.promote_types_of_inputs(diff, x2)
         return torch.round(torch.mul(diff, x2, out=out), out=out).to(x1.dtype)
     return torch.remainder(x1, x2, out=out)
+ master
```
Comment (ogbanugot) [inline ivy/functional/backends/torch/elementwise.py:648]: here as well

**Review by ogbanugot — CHANGES_REQUESTED** [review]

**Comment by Hellenn07** [comment]
Hi,
Thank you for the detailed review. I have resolved the issues. Could you please check it out.
Regards.

File: ivy/functional/backends/numpy/creation.py [hunk 7]
```diff
@@ -137,7 +137,11 @@ def full_like(
     device: str,
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
+ numpy_sort
```
Comment (ogbanugot) [inline ivy/functional/backends/numpy/creation.py:140]: Hi! There seem to still be a few missed lines with unwanted text. Please kindly remove these lines and commit the changes. Thanks!

File: ivy/functional/backends/numpy/creation.py (hunk 1 above)
Comment (ogbanugot) [inline ivy/functional/backends/numpy/creation.py:144]: see here

File: ivy/functional/backends/torch/creation.py [hunk 8]
```diff
@@ -247,7 +247,11 @@ def full_like(
     device: torch.device,
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
+numpy_sort
```
Comment (ogbanugot) [inline ivy/functional/backends/torch/creation.py:250]: here

**Review by ogbanugot — CHANGES_REQUESTED** [review]

**Comment by ogbanugot** [comment]
Also, can you please link the corresponding issue by adding a comment ‘Close #Issue_number’, as explained [here](https://lets-unify.ai/ivy/contributing/1_the_basics.html#todo-list-issues)”

**Comment by ogbanugot** [comment]
Also, can you please write tests for the functions you have added, kindly use the docs on tests [here](https://lets-unify.ai/ivy/deep_dive/15_ivy_tests.html) and [here](https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html). Thanks for your effort so far!

**Comment by Hellenn07** [comment]
#5067

**Comment by Hellenn07** [comment]
#3577

**Comment by Hellenn07** [comment]
Hey, I have committed the changes as requested. I directly commented on the PR but I can't see it linked to my PR yet. I can see we are getting somewhere. I highly appreciate your help.

**Review by ogbanugot — CHANGES_REQUESTED** [review]
Hey! @Hellenn07 sorry for the late response. I'm looking through and there are still issues with the pr with respect to the merge that you did. Some lines of code have been removed and some unwanted lines of code are in the wrong places. If you are unable to fix these, I would suggest that you close this pr and reopen another one that is much cleaner than this one. 
First you should update your fork's master branch by using the `Sync fork` option. 
Next create a new branch from `master` you can use the command `git checkout -b branch-name`. 
Add your contribution to that new branch, commit them and push the changes. You can then open a pr from that branch and request a review from me. I think this would be the fastest approach to take. Otherwise, if you wish to continue working on this branch you should take some time to go through the files and make the appropriate changes.

**Pull request closed without merging**